cmd/agent-js-worker/package.json

```diff
@@ -44,7 +44,7 @@
     "webpack-shell-plugin": "0.5.0"
   },
   "engines": {
-    "node": ">=12.14",
+    "node": ">=12.13",
```

Comment:
Version is decreased from 14 tp 13. Is this intentional ?

Reply:
yes